gradle/scripts/lib/java-rpc-proto.gradle

```diff
@@ -8,8 +8,8 @@ buildscript {
 }
 
 configure(projectsWithFlags('java')) {
-    // Add protobuf/gRPC support if there is src/*/proto
-    if (project.ext.hasSourceDirectory('proto')) {
+    // Add protobuf/gRPC support if there is src/*/proto, gen-src/*/proto
+    if (project.ext.hasSourceDirectory('proto') || project.ext.hasGenerated('proto')) {
```

Comment:
Could you explain why you need this? This will not work if a user ran `./gradlew clean`, which removes the `gen-src` directory.

Reply:
Oops... Looking at the code, it is not necessary. I will remove it.

Reply:
Fixed 44dfcbd